pkg/controller/component/helper.go

```diff
@@ -504,3 +504,36 @@ func updatePod(clientSet kube.ClientSet, newPod, oldPod *corev1.Pod) error {
 func isPending(pod *corev1.Pod) bool {
 	return pod.Status.Phase == corev1.PodPending
 }
+
+// 1. new pvc is not nil, old pvc is not nil, update storage
+func syncPVC(
+	component v1alpha1.NebulaClusterComponent,
+	pvcClient kube.PersistentVolumeClaim) error {
+	replicas := int(component.ComponentSpec().Replicas())
+	volumeClaims, err := component.GenerateVolumeClaim()
+	if err != nil {
+		return err
+	}
+	for _, volumeClaim := range volumeClaims {
+		for i := 0; i < replicas; i++ {
+			pvcName := fmt.Sprintf("%s-%s-%d", volumeClaim.Name, component.GetName(), i)
+			oldPVC, err := pvcClient.GetPVC(component.GetNamespace(), pvcName)
+			if err != nil {
+				if !apierrors.IsNotFound(err) {
+					return err
+				}
+			}
+			if oldPVC == nil {
+				return nil
```

Comment:
If the first one is nil, then other volumeClaim and replicas do not need to be processed?